data/indicator_coefficient_importer/indicator_coefficient_importer.py

```diff
@@ -96,20 +115,15 @@ def main(file: Path, indicator_code: str, year: int):
             indicator_id = cursor.fetchone()[0]
         # add admin region ID to dataframe so we can insert all rows at once
         # without having to query for the IDs every time
-        admin_region_ids = get_admin_region_ids_from_countries(
-            conn, list(data.country.unique())
-        )
+        admin_region_ids = get_admin_region_ids_from_countries(conn, list(data.country.unique()))
         # same with material ID
-        material_ids = get_material_ids_from_hs_codes(
-            conn, data.hs_2017_code.astype(str).to_list()
-        )
+        material_ids = get_material_ids_from_hs_codes(conn, data.hs_2017_code.astype(str).to_list())
 
     data = pd.merge(data, admin_region_ids, on="country", how="left")
     data = pd.merge(data, material_ids, on="hs_2017_code", how="left")
     data["indicatorId"] = indicator_id
-    data_to_insert = data[
-        ["value", "year", "adminRegionId", "indicatorId", "materialId"]
-    ]
+    data = data.drop_duplicates(subset=["value", "year", "adminRegionId", "indicatorId", "materialId"])
```

Comment:
a comment of why we drop duplicates and where the dupes come from would be nice for our future selfs